model/project_vars.go

```diff
@@ -66,6 +68,61 @@ func FindOneProjectVars(projectId string) (*ProjectVars, error) {
 	return projectVars, nil
 }
 
+// FindMergedProjectVars merges vars from the target project's ProjectVars and its parent repo's vars
+func FindMergedProjectVars(projectID string) (*ProjectVars, error) {
+	project, err := FindOneProjectRef(projectID)
+	if err != nil {
+		return nil, errors.Wrapf(err, "problem getting project '%s'", projectID)
+	}
+	if project == nil {
+		return nil, errors.New(fmt.Sprintf("project '%s' does not exist", projectID))
+	}
+
+	projectVars, err := FindOneProjectVars(projectID)
```

Comment:
Just in case the projectID passed into FindMergedProjectVars is a mutable identifier, should use project.Id here to be safe